internal/osutils/autostart/autostart_windows.go

```diff
@@ -0,0 +1,71 @@
+package autostart
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/go-ole/go-ole"
+	"github.com/go-ole/go-ole/oleutil"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/logging"
+)
+
+var startupPath = filepath.Join(os.Getenv("USERPROFILE"), "AppData", "Roaming", "Microsoft", "Windows", "Start Menu", "Programs", "Startup")
+
+func (a *App) Enable() error {
+	if a.IsEnabled() {
+		return nil
+	}
+
+	// ALWAYS errors with "Incorrect function", which can apparently be safely ignored..
+	ole.CoInitializeEx(0, ole.COINIT_APARTMENTTHREADED|ole.COINIT_SPEED_OVER_MEMORY)
+
```

Comment:
I'm not sure, but it may be appropriate to add `defer ole.CoUninitialize()`.
https://pkg.go.dev/github.com/go-ole/go-ole#CoUninitialize